src/pages/setpassword/SetPasswordPageProps.js

```diff
@@ -0,0 +1,38 @@
+import PropTypes from 'prop-types';
+
+export default {
```

Comment:
This file should be named using `camelCase`, use the `somethingPropTypes` convention, and be imported with the same name like this

```js
// setPasswordPagePropTypes.js
import setPasswordPagePropTypes from '../blah';
```